demo/README.md

```diff
@@ -35,7 +35,13 @@ Debug config can be like:
 }
 ```
 
-# Debugging test with chrome devtools
+*In case of windows `node_modules/.bin` folder contains executable files
+ that are not javaScript programs so they cannot be launched with `node`
+ interpreter instead one can use direct path to javaScript file, e.g. 
+ `node_modules/suitest-js-api/lib/testLauncher/index.js` and in case of 
```

Comment:
What if we create launcher.js file in root and it will imports /lib/testLauncher/index.js, so user can use shorter path node_modules/suitest-js-api/launcher.js ?

Reply:
Maybe can it be better to move it to bin folder?, like it is done in all other projects

Reply:
Do you mean creating bash/powershell scripts which will run launcher?

Reply:
I think it good idea